packages/site/src/content/docs/rules/ts/debuggerStatements.mdx

```diff
@@ -0,0 +1,53 @@
+---
+description: "Debugger statements should not be used in production code."
+title: "debuggerStatements"
+---
+
+import { Tabs, TabItem } from "@astrojs/starlight/components";
+import { RuleEquivalents } from "~/components/RuleEquivalents";
+import RuleSummary from "~/components/RuleSummary.astro";
+
+<RuleSummary plugin="ts" rule="debuggerStatements" />
+
+The `debugger` statement causes the JavaScript runtime to pause execution and start a debugger if one is available, such as when browser developer tools are open.
+While this can be useful during development, it should not be left in production code as it can cause unexpected pauses in execution.
+
+## Examples
+
+<Tabs>
+<TabItem label="❌ Incorrect">
+
+```ts
+function processData(data) {
+	debugger;
+	return data.map((item) => item * 2);
```

Comment:
[Docs] Here and in the other code snippet, let's go with no `noImplicitAny` violations:

```suggestion
function doubleValues(values: number[]) {
	debugger;
	return values.map((value) => value * 2);
```